CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 #CHANGE LOG
 
+##Version 0.9.50
+
++ DSL Improvements
+  + Improved Documentation Style - The infile documentation of flows and operations has been changed to be more structured.
+	+ System Properties Files - System properties files now end with the .prop.sl extension and their syntax has changed.
```

Comment:
SP changes should on the same level with `Improved Documentation Style` right? And maybe mention that not only the SP file syntax changes bu the access style